app/components/chart/daily.js

```diff
@@ -33,6 +33,7 @@ var Daily = React.createClass({
   propTypes: {
     bgPrefs: React.PropTypes.object.isRequired,
     chartPrefs: React.PropTypes.object.isRequired,
+    timePrefs: React.PropTypes.object.isRequired,
```

Comment:
Simple question - so timePrefs were part of chartPrefs but have been now broken out - what was this to achieve? Just curious more than anything